From d7f44a8c1e339e31995a8dd6e0361853cf912bf1 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Wed, 30 Dec 2009 01:03:13 +0200
Subject: [PATCH] configure: use libgif and not ungif

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 configure |   10 ++++++----
 1 files changed, 6 insertions(+), 4 deletions(-)

diff --git a/configure b/configure
index 042db98..290c17a 100755
--- a/configure
+++ b/configure
@@ -5001,13 +5001,13 @@ echo "${ECHO_T}$ac_cv_header_gif_lib_h" >&6
 
 fi
 if test $ac_cv_header_gif_lib_h = yes; then
-  echo "$as_me:$LINENO: checking for DGifOpen in -lungif" >&5
-echo $ECHO_N "checking for DGifOpen in -lungif... $ECHO_C" >&6
+  echo "$as_me:$LINENO: checking for DGifOpen in -lgif" >&5
+echo $ECHO_N "checking for DGifOpen in -lgif... $ECHO_C" >&6
 if test "${ac_cv_lib_ungif_DGifOpen+set}" = set; then
   echo $ECHO_N "(cached) $ECHO_C" >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lungif  $LIBS"
+LIBS="-lgif  $LIBS"
 cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
@@ -5068,8 +5068,9 @@ echo "${ECHO_T}$ac_cv_lib_ungif_DGifOpen" >&6
 if test $ac_cv_lib_ungif_DGifOpen = yes; then
   cat >>confdefs.h <<\_ACEOF
 #define HAVE_UNGIF 1
+#define HAVE_GIF 1
 _ACEOF
- LIBS="$LIBS -lungif"
+ LIBS="$LIBS -lgif"
 else
   echo "$as_me:$LINENO: checking for DGifOpen in -lgif" >&5
 echo $ECHO_N "checking for DGifOpen in -lgif... $ECHO_C" >&6
@@ -5138,6 +5139,7 @@ echo "${ECHO_T}$ac_cv_lib_gif_DGifOpen" >&6
 if test $ac_cv_lib_gif_DGifOpen = yes; then
   cat >>confdefs.h <<\_ACEOF
 #define HAVE_UNGIF 1
+#define HAVE_GIF 1
 _ACEOF
  LIBS="$LIBS -lgif"
 fi
-- 
1.6.4.2

